libheif/plugins/encoder_aom.cc

```diff
@@ -162,6 +163,7 @@ static const char* kParam_alpha_min_q = "alpha-min-q";
 static const char* kParam_alpha_max_q = "alpha-max-q";
 static const char* kParam_lossless_alpha = "lossless-alpha";
 static const char* kParam_auto_tiles = "auto-tiles";
+static const char* kParam_intra_block_copy = "intra-block-copy";
```

Comment:
It would be good to name this parameter "enable-intrabc" to match the name used in libaom. On the other hand, I am not sure if libheif has tried to make these parameter names match the names used in libaom, so this may not be worth the trouble.

Reply:
Yes, I believe this makes sense. Parameters that correspond 1:1 to codec parameters should be identical to reduce the confusion and cognitive load.

Reply:
Thanks for the comment, Dirk. This will require a corresponding change to libvips (see https://github.com/libvips/libvips/pull/4284) to use the new parameter name. During the transitional period libheif can support both the old and new parameter names.